#!/usr/bin/env python
PACKAGE = "pioneer_mrs"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# gen.add(name, type, level, description, [default, min, max])

gen.add("formation_offset", double_t, 0, "Add offset to the formation center", 0, 0, 10)

exit(gen.generate(PACKAGE, "algorithm", "Formation"))
